duckduckgo/Android
# Update to autoconsent 3.0.4

Task/Issue URL: https://app.asana.com/0/1201844467387842/1203288491490642/f

### Description
Update to autoconsent 3.0.4 for rule improvements.

### Steps to test this PR
Smoke test


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203301241108850
